src/functions/simpleSum.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  Sketch,
+  GeoprocessingHandler,
+  Polygon,
+  MultiPolygon,
+  ReportResult,
+  SketchCollection,
+  DefaultExtraParams,
+  toNullSketch,
+  rekeyMetrics,
+  getFirstFromParam,
+  Metric,
+  sortMetrics,
+  genTaskCacheKey,
+  GeoprocessingRequestModel,
+} from "@seasketch/geoprocessing";
+
+import project from "../../project/projectClient.js";
+import awsSdk from "aws-sdk";
+
+export async function simpleSum(
+  sketch:
+    | Sketch<Polygon | MultiPolygon>
+    | SketchCollection<Polygon | MultiPolygon>,
+  extraParams: DefaultExtraParams = {},
+  request?: GeoprocessingRequestModel<Polygon | MultiPolygon>
```

Comment:
Geoprocessing functions now have access to the original request parameters sent by the report client.  This allows the sketch URL to be passed on to the workers.